Core/NoOpRunner.Core/Shapes/StaticShapes/ImpassablePlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NoOpRunner.Core.Entities;
+using NoOpRunner.Core.Enums;
+
+namespace NoOpRunner.Core.Shapes
+{
+    public class ImpassablePlatform : StaticShape
+    {
+
+        public ImpassablePlatform(int centerPosX, int centerPosY, int bottomPosY, int topPosY) : base(centerPosX, centerPosY)
+        {
+
+        }
+
+        public override bool CanOverlap(BaseShape other)
+        {
+            return false;
+        }
+
+        public override void OnCollision(BaseShape other)
+        {
+
```

Comment:
**[Nitpick]** Add a `// Do nothing` comment or similar to indicate that this is intentional, and not missing implementation

Reply:
Fixed